packages/markers/src/browser/problem/problem-decorator.ts

```diff
@@ -129,11 +129,19 @@ export class ProblemDecorator implements TreeDecorator {
         const icon = this.getOverlayIcon(marker);
         const color = this.getOverlayIconColor(marker);
         const priority = this.getPriority(marker);
+        //const isResource = Array.from(this.problemManager.getUris()).some(m => m === marker.uri);
+        //const problemStat = this.problemManager.getProblemStat();
```

Comment:
You can remove the following lines, it fails the build due to a missing whitespace.

```
lerna ERR! /home/evinfug/workspaces/omar-theia/packages/markers/src/browser/problem/problem-decorator.ts
lerna ERR!   132:9  error  Expected exception block, space or tab after '//' in comment  spaced-comment
lerna ERR!   133:9  error  Expected exception block, space or tab after '//' in comment  spaced-comment
```

Reply:
On a side note, you can enable [github actions](https://github.com/features/actions) on your repository through the settings to enable CI (so you can verify if your pull-requests build and tests pass):

![image](https://user-images.githubusercontent.com/40359487/117998741-2c5ce200-b312-11eb-91f0-cc51d5319b57.png)